diff -uraN GameHub-0.16.1-6-dev/data/com.github.tkashkin.gamehub.changelog.xml omv-GameHub-0.16.1-6-dev/data/com.github.tkashkin.gamehub.changelog.xml
--- GameHub-0.16.1-6-dev/data/com.github.tkashkin.gamehub.changelog.xml	1970-01-01 01:00:00.000000000 +0100
+++ omv-GameHub-0.16.1-6-dev/data/com.github.tkashkin.gamehub.changelog.xml	2021-10-02 23:37:24.000000000 +0200
@@ -0,0 +1 @@
+d20dfaa0823affae365c9d9b2067453ca4b9b5ec
diff -uraN GameHub-0.16.1-6-dev/data/com.github.tkashkin.gamehub.changelog.xml.REMOVED.git-id omv-GameHub-0.16.1-6-dev/data/com.github.tkashkin.gamehub.changelog.xml.REMOVED.git-id
--- GameHub-0.16.1-6-dev/data/com.github.tkashkin.gamehub.changelog.xml.REMOVED.git-id	2021-10-02 23:37:24.000000000 +0200
+++ omv-GameHub-0.16.1-6-dev/data/com.github.tkashkin.gamehub.changelog.xml.REMOVED.git-id	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-d20dfaa0823affae365c9d9b2067453ca4b9b5ec
